console/console-init/ui/src/modules/address-space/dialogs/CreateMessagingProject/CreateMessagingProject.tsx

```diff
@@ -218,6 +218,7 @@ const CreateMessagingProject: React.FunctionComponent<ICreateMessagingProjectPro
       title="Create an Instance"
       steps={steps}
       onSave={handleSave}
+      height={400}
```

Comment:
Can we instead create a const variable for specifying the height and width of all the wizards, instead of hard coding here?